Project Programmer fails to compile

**Description**
Building the Programmer project with error:
```
Programmer\TargetTester.cpp(448,16): error C2039: 'status': is not a member of 'ETarget'
Programmer\Programmer.hpp(21,7): message : see declaration of 'ETarget'
```
**Repro steps**
1) Build solution

**Environment**
Visual Studio 2022
C++ ver: /std:c++latest
Tested commit: 211c74d1f14c6585e94b28bed1e976f73730ffb9

--- a/Programmer.cpp
+++ b/Programmer.cpp
@@ -149,7 +149,7 @@ void IProgrammerStrategy::erase_write(uint32_t address, const std::span<const st
 /* ETarget */
 
 ETarget::ETarget(const Protocol::Status status)
-	: Exception(), _status(status)
+	: Exception(), status(status)
 {
 	switch (status) {
 		case Protocol::STATUS_INV_OP:

--- a/Programmer.hpp
+++ b/Programmer.hpp
@@ -21,7 +21,7 @@
 class ETarget : public Exception {
 	public:
 		ETarget(const Protocol::Status status);
-		const Protocol::Status _status;
+		const Protocol::Status status;
 };
 
 struct ProgrammerDescriptor {

